src/components/n-file-select/n-file-select-view-model.ts

```diff
@@ -76,6 +76,19 @@ export class NFileSelectViewModel extends ComponentViewModel
     }
 
 
+    protected override onCreate(): void
+    {
+        super.onCreate();
+
+        given(this._mimeTypesList, "mimeTypesList")
+            .ensure(t => t.split(",").every(mime =>
+            {
+                mime = mime.trim();
+                return (mime.startsWith(".") && !mime.contains("/"))
```

Comment:
Why would mime type start with `.`?

Reply:
It could be file extension or mime type https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#unique_file_type_specifiers